### PR TITLE
report: remove pr_curve mask

### PR DESCRIPTION
**Pull request by doron-st**

### CODE REVIEW

File: ugvc/reports/report_utils.py [hunk 1]
```diff
@@ -178,12 +178,13 @@ def plot_performance(
                 ax_score_row[0].legend(loc="upper right")
 
                 best_f1 = perf.f1.max()
-                f1_range = self.__get_range_from_gap(best_f1)
-                ax_pr_row[j].set_xlim(f1_range)
-                ax_pr_row[j].set_ylim(f1_range)
-                max_score = perf[perf.f1 >= f1_range[0]][self.score_name].max()
-                ax_score_row[j].set_xlim([0, max_score])
-                ax_score_row[j].set_ylim(f1_range)
+                if not math.isnan(best_f1):
```
Comment (doron-st) [inline ugvc/reports/report_utils.py:181]: could fail

File: ugvc/reports/report_utils.py [hunk 2]
```diff
@@ -178,12 +178,13 @@ def plot_performance(
...
+                    f1_range = self.__get_range_from_gap(best_f1)
+                    ax_pr_row[j].set_xlim(f1_range)
+                    ax_pr_row[j].set_ylim(f1_range)
+                    max_score = max(perf[perf.f1 >= f1_range[0]][self.score_name].max(), 0.01)
```
Comment (doron-st) [inline ugvc/reports/report_utils.py:185]: robust to 0 range warning

File: ugvc/reports/report_utils.py [hunk 3]
```diff
@@ -195,8 +196,9 @@ def plot_performance(
         fig_score.tight_layout()
         plt.show()
 
-    def __get_range_from_gap(self, metric):
-        gap = 1 - metric
+    @staticmethod
+    def __get_range_from_gap(metric):
+        gap = max(1 - metric, 0.01)
```
Comment (doron-st) [inline ugvc/reports/report_utils.py:201]: robust to 0 range warning

File: ugvc/reports/report_utils.py [hunk 4]
```diff
@@ -458,10 +460,10 @@ def __calc_performance(self, data: pd.DataFrame) -> tuple[dict, pd.DataFrame]:
         wrong_allele = ((error_type == ErrorType.WRONG_ALLELE) & (d["filter"] == "PASS")).sum()
         filtered_true = fn - missing_candidates - hom_to_het - het_to_hom - wrong_allele
 
-        recall = get_recall(fn, tp, 0)
-        max_recall = get_recall(missing_candidates, (tp + fn - missing_candidates), 0)
-        precision = get_precision(fp, tp, 0)
-        f1 = get_f1(recall, precision, 0)
+        recall = get_recall(fn, tp, np.nan)
```
Comment (doron-st) [inline ugvc/reports/report_utils.py:463]: unrelated bugfix

**Review by ilyasoifer — APPROVED** [review]

**Pull request merged**